### PR TITLE
[Proof of Concept / PitchSpeller] Dumb down FlowNetwork construction

### PR DESCRIPTION
**Pull request by jsbean**
This PR is an approach to #185. It reimplements the current FP-style `FlowNetwork` masking with a dumber, imperative style.

In this case, we don't start with a near-complete graph only to mask away edges. Instead, we build up the graph, only connecting the nodes how we want 'em.

### CODE REVIEW

**Comment by jsbean** [comment]
The test failures are the same seven as #176.

File: Tests/SpelledPitchTests/PitchSpellerTests.swift [hunk 1]
```diff
@@ -352,11 +359,33 @@ class PitchSpellerTests: XCTestCase {
...
+        switch self {
+        case .source:
+            return "source"
+        case .sink:
+            return "sink"
+        case .internal(let index):
+            return "\(index)"
+        }
+    }
+}
```
Comment (bwetherfield) [inline Tests/SpelledPitchTests/PitchSpellerTests.swift]: These are great! Let's actually merge some of these changes in... ?

Comment (jsbean) [reply Tests/SpelledPitchTests/PitchSpellerTests.swift]: Just want the `CustomStringConvertible` bits?

File: Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift [hunk 2]
```diff
@@ -116,37 +73,66 @@ extension PitchSpeller {
...
+                    let pcA = pitch(a)!.class
+                    let pcB = pitch(b)!.class
+                    let pair = UnorderedPair(Cross(pcA,a.tendency),Cross(pcB,b.tendency))
+                    internalEdgeLookup[pair].map { weight in
+                        flowNetwork.insertEdge(from: a, to: b, weight: weight)
+                    }
+                default:
+                    continue
+                }
+            }
```
Comment (bwetherfield) [inline Sources/SpelledPitch/PitchSpeller/Wetherfield/Wetherfield.swift]: This all looks on the money to me 👍

File: Tests/SpelledPitchTests/PitchSpellerTests.swift (hunk 1 above)
Comment (bwetherfield) [reply Tests/SpelledPitchTests/PitchSpellerTests.swift]: Yeah, I have a feeling those will be very handy when tests fail

Comment (bwetherfield) [reply Tests/SpelledPitchTests/PitchSpellerTests.swift]: Do they show up in tests?

Comment (jsbean) [reply Tests/SpelledPitchTests/PitchSpellerTests.swift]: When they fail, yes. Without the `CustomStringConvertible` its like 30 pages long 😭 .

Comment (jsbean) [reply Tests/SpelledPitchTests/PitchSpellerTests.swift]: K, I'll do another PR with them in the right place.

Comment (bwetherfield) [reply Tests/SpelledPitchTests/PitchSpellerTests.swift]: And yes, I have read through those failures in times of bemusement!

Comment (bwetherfield) [reply Tests/SpelledPitchTests/PitchSpellerTests.swift]: I usually end up copying into `vim` and diving into the world of selective deletion, but this is the right way to go about things (note to self)

**Pull request closed without merging**